<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>cetserver</class>
 <widget class="QMainWindow" name="cetserver">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>283</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CET Server</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="stopServerButton">
    <property name="geometry">
     <rect>
      <x>197</x>
      <y>52</y>
      <width>170</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Stop Server</string>
    </property>
   </widget>
   <widget class="QToolButton" name="openDatabaseButton">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>19</y>
      <width>98</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Open Database</string>
    </property>
   </widget>
   <widget class="QPushButton" name="startServerButton">
    <property name="geometry">
     <rect>
      <x>24</x>
      <y>52</y>
      <width>171</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Start Server</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="statusText">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>90</y>
      <width>331</width>
      <height>127</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>400</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionExit"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>openDatabaseButton</sender>
   <signal>clicked()</signal>
   <receiver>cetserver</receiver>
   <slot>openDatabaseButtonHandler()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>78</x>
     <y>63</y>
    </hint>
    <hint type="destinationlabel">
     <x>237</x>
     <y>62</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>stopServerButton</sender>
   <signal>clicked()</signal>
   <receiver>cetserver</receiver>
   <slot>stopServerButtonHandler()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>286</x>
     <y>101</y>
    </hint>
    <hint type="destinationlabel">
     <x>386</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>startServerButton</sender>
   <signal>clicked()</signal>
   <receiver>cetserver</receiver>
   <slot>startServerButtonHandler()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>146</x>
     <y>102</y>
    </hint>
    <hint type="destinationlabel">
     <x>11</x>
     <y>146</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>openDatabaseButtonHandler()</slot>
  <slot>stopServerButtonHandler()</slot>
  <slot>startServerButtonHandler()</slot>
 </slots>
</ui>
